src/figuresClasses.ts

```diff
@@ -1,19 +1,98 @@
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+type Color = 'red' | 'green' | 'blue';
```

Comment:
Better to make `enum` here also

Reply:
But why enum is better than type? We don`t know what exactly color Class will get. In tests, we also pass a string value as the first argument // const circle = new Circle('red', 6) not  //const circle = new Circle(Colors.Red, 6);
For me 'type' is easier and shorter)